history-service/models/history.js

```diff
@@ -0,0 +1,22 @@
+import mongoose from "mongoose"
+
+const historySchema = new mongoose.Schema({
+    username: {
+        type: String,
+        required: true,
+    },
+    matchUsername: {
```

Comment:
what does matchUsername refer to? the other user?

Reply:
yup, refers to other user